### DISCUSSION TITLE
Sample Facebook data?

### DISCUSSION THREAD

**Discussion by MinaBasem** [Kubis10/CounterForMessenger#54] [Q&A] [answered]
Hello Kubis,

I was wondering if you would by any chance have sample data that can be used for this project since Facebook's information download service seems to be down.

I am willing to contribute to the UI but obtaining the data is an obstacle right now.

If you do by any chance I'd love to have it.

Thank you.

**Comment by Kubis10** [comment]
Hi, I have only my own Facebook data, but I can fabricate some test data for this project

**Comment by Kubis10** [comment, marked as answer]
@MinaBasem now you have some sample data, I hope it will be enough (Put your name as John Doe)

**Reply by MinaBasem** [reply]
Thank you! Will see what I can do with the UI modification and let you know. Again, thanks a lot.
